test/workbox-precaching/integration/test-precache-and-update.js

```diff
@@ -52,10 +52,15 @@ describe(`[workbox-precaching] Precache and Update`, function() {
     requestCounter = global.__workbox.server.startCountingRequests();
 
     // Request the page and check that the precached assets weren't requested from the network.
+    // Include the default ignoreURLParametersMatching query parameters.
     await global.__workbox.webdriver.get(`${baseURL}index.html`);
+    await global.__workbox.webdriver.get(`${baseURL}index.html?utm_source=test`);
+    await global.__workbox.webdriver.get(`${baseURL}index.html?^fbclid$=test`);
```

Comment:
Is this URL supposed to have the `^` and `$` RegExp characters in it? (Same question for line 63.)

Reply:
BTW, if this was a mistake and the tests are still passing, maybe update the tests to also check the inverse?